site/src/routes/embedding-knn/AgreementClusterStaticPlot.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="ts">
+  import { browser } from "$app/environment";
+
+  export let data: any[] = [];
+  let selected: string = "normoc";
+  let api_url: string =
+    "/api/v1/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-static/v1";
+
+  $: names = data.map(async (d) => {
+    if (browser) {
+      let resp = await fetch(
+        "/api/v1/data/processed/birdclef-2022/birdnet-embeddings-with-neighbors-static/v1/" +
+          d["ego_primary_label"] +
+          "/distances.png"
+      );
+      if (resp.status != 404) {
+        return d["ego_primary_label"];
+      }
+    }
+  });
```

Comment:
We don't actually need to check if the images exists or not -- we just trust the manifest (agreement.json) to tell us about all the files.

I'd personally just simplify this to `$: names = data.map(d => d.ego_primary_label)`.

Reply:
The agreement.json includes an entry for every single bird species and not just the ones which have images

Reply:
Oh, you're right. I could have sworn I wrote out a manifest but I suppose that isn't true. We should probably go back to the first 20 entries then -- sorry.